homeassistant/components/postnl/sensor.py

```diff
@@ -58,7 +58,7 @@ class PostNLSensor(Entity):
     def __init__(self, api, name):
         """Initialize the PostNL sensor."""
         self._name = name
-        self._attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
+        self._attributes = {ATTR_ATTRIBUTION: ATTRIBUTION, "shipments": []}
```

Comment:
Should we extract the `"shipments"` string into a module level constant, since we're using it in three places and to avoid typos?

Reply:
See the PR mentioned in the description. That one indeed does that. This is a sort of in between PR to make sure that at least this component works, as it has been broken for almost a week I believe.

Reply:
Unless you already also want it to be changed over here already, I am happy to apply the change for you.

Reply:
Ok, good! Let's save it for the other PR.